### PR TITLE
Replace TraceDroid-1.1.jar with jcenter dependency

### LINKED ISSUE #16: SSLHandshakeException / SSLProtocolException: handshake aborted: Failure in SSL library, usually a protocol error (Android 4)
# Environment
- Android 4.1.2
- Samsung GT-I8190

# How to reproduce
1. Install [`Datenspuren v.1.33.0-Datenspuren-Edition` app](https://github.com/johnjohndoe/CampFahrplan/tree/v.1.33.0-Datenspuren-Edition) (DEBUG or RELEASE)
2. Start app
3. SSL certficate dialog is shown

# Expected behavior
- SSL dialog can be confirmed. Data is fetched.

# Observed behavior
- Failure dialog reappears on confirmation.

Dialog SSL handshake aborted

## Stacktrace

``` java
W  javax.net.ssl.SSLHandshakeException: javax.net.ssl.SSLProtocolException: 
    SSL handshake aborted: ssl=0x40e83050: Failure in SSL library, usually a protocol error
W  error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version 
    (external/openssl/ssl/s23_clnt.c:741 0x40e0276a:0x00000000)
W      at org.apache.harmony.xnet.provider.jsse.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:412)
W      at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.java:299)
W      at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.java:268)
W      at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:160)
W      at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)
W      at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)
W      at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)
W      at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
W      at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
W      at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
W      at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
W      at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
W      at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
W      at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
W      at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
W      at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)
W      at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
W      at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
W      at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
W      at okhttp3.RealCall.execute(RealCall.java:77)
W      at nerd.tuxmobil.fahrplan.congress.net.FetchFahrplanTask.fetch(FetchFahrplan.java:142)
W      at nerd.tuxmobil.fahrplan.congress.net.FetchFahrplanTask.doInBackground(FetchFahrplan.java:93)
W      at nerd.tuxmobil.fahrplan.congress.net.FetchFahrplanTask.doInBackground(FetchFahrplan.java:58)
W      at android.os.AsyncTask$2.call(AsyncTask.java:287)
W      at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:305)
W      at java.util.concurrent.FutureTask.run(FutureTask.java:137)
W      at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:230)
W      at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1076)
W      at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:569)
W      at java.lang.Thread.run(Thread.java:856)

W  Caused by: javax.net.ssl.SSLProtocolException: SSL handshake aborted: ssl=0x40e83050: 
    Failure in SSL library, usually a protocol error
W  error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version 
    (external/openssl/ssl/s23_clnt.c:741 0x40e0276a:0x00000000)
W      at org.apache.harmony.xnet.provider.jsse.NativeCrypto.SSL_do_handshake(Native Method)
W      at org.apache.harmony.xnet.provider.jsse.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:371)
W      ... 29 more
```

### PR DESCRIPTION
**Pull request by grote**
Closes #16

### CODE REVIEW

**Comment by grote** [comment]
I updated the dependency while I was at it.

File: app/build.gradle [hunk 1]
```diff
@@ -132,7 +132,7 @@ dependencies {
     implementation "com.squareup.okhttp3:logging-interceptor:$okhttpVersion"
     implementation "com.github.ligi.snackengage:snackengage-playrate:$snackEngageVersion"
     implementation "de.cketti.mailto:email-intent-builder:$emailIntentBuilderVersion"
-    implementation files("libs/TraceDroid-1.1.jar")
+    implementation "org.ligi:tracedroid:1.4"
```
Comment (johnjohndoe) [inline app/build.gradle]: Please define the version along with the other dependency version as `traceDroidVersion = 1.4`. Force push your branch.

Comment (grote) [reply app/build.gradle]: Done

**Comment by johnjohndoe** [comment]
Thank you! :cat2:

**Pull request merged**